workflow/setup_expt.py

```diff
@@ -182,19 +233,20 @@ def edit_baseconfig(host, inputs):
         extend_dict = {
             "@CASEENS@": f'C{inputs.resens}',
             "@NMEM_ENKF@": inputs.nens,
+            "@DOHYBVAR@": "YES" if inputs.nens > 0 else "NO",
```

Comment:
"--nens" is not a required argument when running setup_expt.py and defaults to "20" if not specified.  I'm uncomfortable having IF statements that rely on nens>0 to mean the user intends on using an ensemble.  I know that this isn't the only time it's used like this in the script, so it might be a big ask.  Not a PR stopper for me, but should be addressed in the future.

Reply:
I'm not sure how this would work between cycled mode and free forecast mode, but is it possible to make the nens argument required for cycled mode only?  

One idea that would require little coding work is to add language to the help description/wiki page to explicitly state that if you're not using ensembles, you need to specify nens=0, something like "help='number of ensemble members, set to 0 for 3DVar'".

Reply:
`nens` is an argument available only in cycled mode.  See https://github.com/NOAA-EMC/global-workflow/blob/a9ab1e753808454d713e3863ac34e911fe65bbe5/workflow/setup_expt.py#L289.  There is no support for ensembles in free-forecast mode.
When running free-forecast mode, `nens` is not in scope.

Reply:
You can see for yourself with:
```
setup_expt.py cycled -h
```
and 
```
setup_expt.py forecast-only -h
```
You will see different arguments are available for the 2 modes.